manifests/templates/cicd/sonarqube.yaml

```diff
@@ -0,0 +1,54 @@
+- apiVersion: cyclone.dev/v1alpha1
+  kind: Stage
+  metadata:
+    name: sonarqube-template
```

Comment:
codescan-template?

Reply:
I'd prefer sonarqube-template, it's more accurate.